src/main/java/core/basesyntax/model/Operations.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.model;
+
+public enum Operations {
+    R,P,S,B;
+
+    public static boolean contains(String value) {
+        try {
+            Operations.valueOf(value.toUpperCase());
+            return true;
+        } catch (IllegalArgumentException e) {
+            return false;
+        }
+    }
```

Comment:
what about the previous comment on this?
please replace this method with `public static Operations getByString(String value)`